trade.md

```diff
@@ -9,14 +9,62 @@ gcam-version: v5.3
 International trade in most commodities in GCAM is done by one of three methods: (1) Heckscher-Ohlin or single global markets, (2) global trade with regionally-differentiated markets with Armington-like preferences between domestic and imported commodities, and (3) fixed interregional trade.  Other approaches for trade can also be implemented in the GCAM framework (such as [GCAM USA](gcam-usa.html) where logit based decisions are made to facilitating trade between the 50-states).
 
 ## Heckscher-Ohlin
-Commodities such as coal, gas, oil, bio-energy, Corn, Rice, etc are each traded in a single global market.  Each region will see the same global price and independently decide how much each will supply and demand of each commodity given that price.  A region's net trade position is dynamic depending on economics, technical change, demand, growth, resources, etc.  Under this method for trading goods there is no modeled preference for a given region to demand a commodity from any other specific region.
 
-## Armington Style Trade for Primary Ag
-For the primary agriculture commodities such as Corn, Rice, and purpose grown bio-energy we use an Armington style distinction between domestic and imported goods, but not with fixed bilateral imports.    The competition between imports and domestic is governed by a logit.  Imports are from a single global pool that draws from all regions and is also governed by a logit.
+The Heckscher-Ohlin theorem explains trade using factor endowments and predicts that each country produces goods with more intensive use of its abundant factor of production (Peter Debaere, 2003; Vanek, 1968). The empirical use of the Heckscher-Ohlin approach assumes products are homogeneous across sources and traded in a single global market (i.e., fully integrated world market). Markets clear at the world level and each region will see the same global price and independently decide how much each will supply and demand of each commodity given that price. A region's net trade position is dynamic depending on economics, technical change, demand, growth, resources, etc.  Under this method for trading goods there is no modeled preference for a given region to demand a commodity from any other specific region. 
+
+The trade of agricultural and livestock proudcts were mostly modeled using the Heckscher-Ohlin approach in early version of GCAM (e.g., GCAM v4). But it has been updated to the Armington style trade modeling approach for most of the agricultural and livestock proudcts. Fodder_herb is modeled using the Heckscher-Ohlin approach and Fodder_grass is not traded. Also, major energy commodities such as coal, gas, oil, bio-energy, etc. are also traded in a single world market with the Heckscher-Ohlin approach.      
```

Comment:
Please check spelling of "products". Also "FodderHerb" and "FodderGrass"

Reply:
It may confuse people that it says the trade of agriculture and livestock had been HOV in GCAM 4.  Instead meat and dairy were traded using fixed, static net imports for each region matched to base year data.